--- db-5.3.28/dist/aclocal/tls.m4	2013-09-09 17:35:02.000000000 +0200
+++ db-5.3.28/dist/aclocal/tls.m4	2026-05-18 01:30:13.963857395 +0200
@@ -15,7 +15,8 @@
       for ax_tls_defn_keyword in $ax_tls_keywords ""; do
           test -z "$ax_tls_decl_keyword" &&
               test -z "$ax_tls_defn_keyword" && continue
-          AC_TRY_COMPILE([template <typename T>class TLSClass {
+          AC_TRY_COMPILE([#include <cstddef>
+              template <typename T>class TLSClass {
               public: static ] $ax_tls_decl_keyword [ T *tlsvar;
               };
               class TLSClass2 {
